src/ios/AppDelegate+APPLocalNotificationAction.m

```diff
@@ -0,0 +1,43 @@
+/*
+ * AppDelegate+APPLocalNotificationAction.m
+ *
+ * Created by Elli Rego on 10/28/15.
+ *
+ */
+
+ #import "AppDelegate+APPLocalNotificationAction.h"
+
+ @implementation AppDelegate (APPLocalNotificationAction)
+
+/**
+ * Handle notification actions for iOS < 9.
+ */
+ - (void)application:(UIApplication *)application handleActionWithIdentifier:(NSString *)identifier forLocalNotification:(UILocalNotification *)notification completionHandler:(void(^)())completionHandler  
+ {
+    [self application:[UIApplication sharedApplication]
+        handleActionWithIdentifier:identifier
+        forLocalNotification:notification
+        withResponseInfo:nil
+        completionHandler:(void (^)())completionHandler];
+    
+    completionHandler();
```

Comment:
Looks like the handler will called twice (here and in again in `handleActionWithIdentifier`)